### PR TITLE
Retinal Blindness (Diabetic Retinopathy) detection

### PR DESCRIPTION
**Pull request by quantumtechniker**
Problem statement :
Diabetic Retinopathy is a disease with an increasing prevalence and the main cause of blindness among working-age population. The risk of severe vision loss can be significantly reduced by timely diagnosis and treatment. Systematic screening for DR has been identified as a cost-effective way to save health services resources. Automatic retinal image analysis is emerging as an important screening tool for early DR detection, which can reduce the workload associated to manual grading as well as save diagnosis costs and time. Many research efforts in the last years have been devoted to developing automated tools to help in the detection and evaluation of DR lesions. We are interested in automating this predition using deep learning models.

Dataset : [APOTS Kaggle Blindness dataset](https://www.kaggle.com/c/aptos2019-blindness-detection)

I am proposing Deep Learning classification technique using CNN pretrained model [resnet152](https://github.com/pytorch/vision/blob/master/torchvision/models/resnet.py) to classify severity levels of DR ranging from 0 (NO DR) to 4 (Proliferative DR).
This is a collaborative project of team of three where my main work is on developing, training and testing various CNN models along with some secondary work. Deep learning looks promising because already various types of image classification tasks has been performed by various CNN's so, we can rely on DL pretrained models or we can modify some layers if we wish to :)
A GUI based system has been made using Tkinter and used heidiSQL to maintain and store a list of predictions with their patient id and name (which is very risky , the reason we will get to it some time later).
Twilio API have been used to Make SMS connectivity to patients possible in case they are not contactable or accesible (in that case we can also use mail).

### CODE REVIEW

**Comment by quantumtechniker** [comment]
I am Working towards artificial general intelligence (AGI) is a highly ambitious and complex endeavor. AGI refers to a form of artificial intelligence that possesses the ability to understand, learn, and apply knowledge across a wide range of tasks and domains, similar to the general intelligence capabilities of a human being. Here are some key aspects and considerations when working towards AGI:

 AGI aims to create machines that can perform any intellectual task that a human can do, from understanding natural language and recognizing patterns to learning new skills and solving complex problems. It goes beyond narrow AI, which is designed for specific tasks.

Achieving AGI requires expertise in various fields, including machine learning, natural language processing, computer vision, robotics, neuroscience, cognitive psychology, and more. Collaboration across these disciplines is essential.
